sites/website/src/prepare-docs.js

```diff
@@ -78,7 +78,7 @@ function copyMarkdown() {
             const folder = path.dirname(source);
             const dest = path.join(
                 root,
-                `fast-${folder.substr(folder.lastIndexOf("/") + 1)}.md`
+                `fast-${folder.substr(folder.lastIndexOf("/") + 1)}.mdx`
```

Comment:
why can't we use mdx, is that where the `live` syntax for code blocks comes from?

Reply:
@chrisdholt This file will probably have to be manually merged. I suggest you take the master version with my updates and then manually apply the this change, since it's pretty straight forward.